update-affiliated/update_astropy_helpers.py

```diff
@@ -28,6 +29,21 @@
 gh = Github(username, password)
 user = gh.get_user()
 
+HELPERS_UPDATE_MESSAGE_BODY = re.sub('(\S+)\n', r'\1 ', """
+This is an automated update of the astropy-helpers submodule to {0}. This
+includes both the update of the astropy-helpers sub-module, and the
+``ah_bootstrap.py`` file, if needed.
+
+
```

Comment:
Oh, that's a tricky one, and more like a hack than a full understanding. The regex here needed to use ``\S+`` rather than the one I cooked up for the bot with ``\w+``, as the last line is ending with a special \\*\\ character. However somehow using single lines everything ended up in one paragraph without the empty lines.

Reply:
Ok no problem :)